docs/api/doc-blocks/doc-block-colorpalette.mdx

```diff
@@ -46,6 +46,18 @@ import { Meta, ColorPalette, ColorItem } from '@storybook/blocks';
       Apple30: 'rgba(102,191,60,.3)',
     }}
   />
+ <ColorItem
+      title="gradient"
+      subtitle="Grayscale"
+      colors={{
+        Gradient: 'linear-gradient(to right,white,black)',
+      }}
+  />
+ <ColorItem
+      title="gradient"
+      subtitle="Grayscale"
+      colors={['linear-gradient(65deg,white,black)']}
+  />
```

Comment:
@aberllin when you have a moment, can you adjust the spacing to match the rest of the example, as it's a bit off? It's using 4 spaces when it should be 2 spaces instead.

Reply:
Hi @ jonniebigodes , please check again! 
Thank you